timbuctoo-instancev4/src/main/java/nl/knaw/huygens/timbuctoo/rdf/CollectionDescription.java

```diff
@@ -20,6 +20,7 @@ private CollectionDescription(String entityTypeName, String vreName, String rdfU
     this.entityTypeName = entityTypeName;
     this.vreName = vreName;
     this.rdfUri = rdfUri;
+    this.unknown = entityTypeName != null && entityTypeName.endsWith("unknown");
```

Comment:
this would make any collection that ends with unknown an "unknown" collection. also. concepts is now no longer an unknown collection